apps/package.json

```diff
@@ -129,7 +130,6 @@
     "jquery-ui": "^1.12.1",
     "js-cookie": "^2.1.2",
     "js-yaml": "^3.6.0",
-    "json-loader": "0.5.4",
```

Comment:
No replacement was needed for `json-loader`?

Reply:
Good catch. A json parsing error prompted me to remove this. According to https://webpack.js.org/loaders/json-loader/ it's no longer needed:
> ⚠️ Since webpack >= v2.0.0, importing of JSON files will work by default. You might still want to use this if you use a custom file extension. See the [v1.0.0 -> v2.0.0 Migration Guide](https://webpack.js.org/migrate/3/#json-loader-is-not-required-anymore) for more information